src/windows/system.rs

```diff
@@ -407,7 +407,11 @@ impl SystemExt for System {
         get_dns_hostname()
     }
 
-    fn get_version(&self) -> Option<String> {
+    fn get_kernel_version(&self) -> Option<String> {
+        get_system_version()
+    }
+
+    fn get_os_version(&self) -> Option<String> {
```

Comment:
Maybe add a FIXME until it's actually fixed and replace the `get_system_version()` call with `None`. I don't want to return "false" information even if temporary.

Reply:
I added a `// FIXME` comment pointing to the two issues (#415 and #410) 

I also made `get_kernel_version()` return `None`, but I did keep the current functionality in `get_os_version()`, I think we are using that for Windows build numbers right now and it would be nice to have something to keep the same functionality we currently have before we find a solution.

Reply:
heh this also makes the Windows tests fail, I could do a `#[cfg(not(target_os = "windows"))]` but I think we should let the functionality continue and break less code until we find better API calls to use.

Reply:
It's fine to add a `cfg` in the test with a fixme comment.